### PR TITLE
Create 2024-06-14-Tales-From-TCamp.md

### PR DESCRIPTION
**Pull request by laneymangan**

### CODE REVIEW

File: src/_posts/2024-06-14-Tales-From-TCamp.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+excerpt: "Lessons learned from 2024 LA Transportation Camp."
+date: 2024-06-13 19:03:13 +0200
+categories: compiler
+---
+
+Recently, four members of the Compiler team had the opportunity to attend the latest edition of [UCLA’s Transportation Camp LA](https://www.its.ucla.edu/). Shout out
+to Juan Matute and the wonderful team at UCLA Institute for Transportation Studies for bringing together folks from advocacy, academia, and both the public and private sectors on a Saturday
+to explore what transit means to us and our broader community. Each of our team members got to participate in a variety of breakouts and we wanted to share our experiences.
+
+## <img src="/src/assets/team_members/nina-dinh.png" class="align-center w-25" alt="Watercolor illustration of Nina Dinh"> Nina
```
Comment (AnthonyRollins) [inline src/_posts/2024-06-14-Tales-From-TCamp.md:16]: It appears this image isn't showing, so I believe you have to remove the `/src/` as the image for Marie appears to be working

File: src/_posts/2024-06-14-Tales-From-TCamp.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+encourage collaborative efforts between transportation and the arts.
+
+During the session with Phoebe, attendees shared their experiences with creative collaborations in the public transit sector. I was introduced to the work of [Marcela Armas](https://www.marcelaarmas.net/?works=exhaust),
+a Mexican artist whose performance art installations provoke discussions on the environmental and mental effects of cars. I also learned about [Outfront/JC Decaux’s
+#commutersofla transit series](http://www.outfrontjcdecaux.com/), in which they utilize bus shelters for community art installations. Witnessing how art could be seamlessly integrated into transportation
+initiatives in impactful ways has left me feeling inspired to seek out further opportunities for merging art and public transit.
+
+## <img src="/src/assets/team_members/vyki-englert.png" class="align-center w-25" alt="Watercolor illustration of Vyki Englert"> Vyki
```
Comment (AnthonyRollins) [inline src/_posts/2024-06-14-Tales-From-TCamp.md:48]: Remove the `/src/` from link

File: src/_posts/2024-06-14-Tales-From-TCamp.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Together we workshopped:
+- Challenges: this is really wonky, how do you get folks excited about it?
+- Strategy + Tactics: tie TDM to a successful car-free Olympics, find new champions for TDM on the City Council who can get it agendized
+- Potential allies: Move LA, Parking Reform Network, Investing in Place, LA28
+
+We agreed that a key need is developing shared messaging around why this is important and can help us have conversations with colleagues, advocates, allies, and champions.
+While we didn’t walk out of the room with any specific homework, I’m excited to continue these conversations with the folks I met last week. TDM is an important tool on
+our path to becoming a climate-resilient city.
+
+## <img src="/src/assets/team_members/scott-frazier.png" class="align-center w-25" alt="Watercolor illustration of Scott Frazier"> Scott
```
Comment (AnthonyRollins) [inline src/_posts/2024-06-14-Tales-From-TCamp.md:72]: Remove the `/src/` from link

File: src/_posts/2024-06-14-Tales-From-TCamp.md (hunk 1 above)
Comment (laneymangan) [reply src/_posts/2024-06-14-Tales-From-TCamp.md]: Hm, so odd...I went back and forth between /src/ and no /src/ and it finally worked when I removed /src/ for all but Marie but it doesn't appear to be working anymore. I went ahead and removed it again but it's not showing up for me. Perhaps @angela-tran might have an idea?

Comment (laneymangan) [reply src/_posts/2024-06-14-Tales-From-TCamp.md]: wait....now its showing up. @AnthonyRollins does it look right to you?

**Comment by laneymangan** [comment]
Another blog post ready for review/approval @compilerla/engineering :)

**Review by angela-tran — APPROVED** [review]
> Another blog post ready for review/approval @compilerla/engineering :)

Looks good to me @laneymangan

**Comment by laneymangan** [comment]
@AnthonyRollins I think the photo issue is resolved, so this should hopefully be good to approve. Angela approved as well

**Comment by angela-tran** [comment]
@laneymangan You can also go ahead and mark this pull request as "ready for review." That will take the pull request out of "Draft"

**Review by AnthonyRollins — APPROVED** [review]

**Pull request merged**